.circleci/should_build_docker.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -eo pipefail
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+PROJECT_DIR=$SCRIPT_DIR/..
+
+# Look for updated Dockerfiles in the commit
+cd "$PROJECT_DIR"
+DOCKER_FILES=$(find $(git diff-tree --no-commit-id --name-only -r HEAD) -name "*.Dockerfile")
+
```

Comment:
This probably isn't what you want because authors can include more than 1 commit in a PR. Take a look at the code in git-checks.sh if you want to ensure that no dockerfiles were changed in a particular PR.

Reply:
Also this step could be significantly simplified without the find as
```suggestion
DOCKER_FILES=$(git diff-tree --no-commit-id --name-only -r HEAD -- "*.Dockerfile")
```

Reply:
Done. Updated the condition check to look at all commits in the PR.